### ISSUE TITLE
Purpose/goal for Bank Of Canada does not make use of capabilities FIBO has for that

### ISSUE THREAD

**Issue by rivettp** [edmcouncil/fibo#893]
As an exemplar the following property of https://spec.edmcouncil.org/fibo/ontology/FBC/FunctionalEntities/NorthAmericanEntities/CARegulatoryAgencies/BankOfCanada should instead be modeled as an instance of https://spec.edmcouncil.org/fibo/ontology/BE/LegalEntities/LegalPersons/PublicPurpose. Or alternatively I'd be happy to get rid of the whole set of Goals and Objectives ontologies/classes, which IMO are of little value except when modeling an investment fund.

```
<fibo-fnd-utl-av:explanatoryNote>The Bank of Canada&apos;s overall goal is to promote a stable and efficient financial system in Canada. The focus on the financial system as a whole parallels the
Bank&apos;s approach to monetary policy, which focuses on the entire economy.
The Bank provides liquidity to the financial system, gives policy advice to the federal government on the design and development of the system, oversees major clearing and settlement systems, and provides banking services to these systems and their participants.
```

**Comment by ElisaKendall** [comment]
The Bank of Canada is not a public purpose, it is an instrumentality.  It may _**have**_ a public purpose, in which case we could add a triple to it for hasObjective PublicPurpose, but I would not make it an individual of type PublicPurpose.  

None of these individuals were ever intended to be exhaustively complete by any stretch of the imagination ...

**Comment by rivettp** [comment]
Yes I was not for a moment proposing that Bank of Canada have type Purpose, but that its purpose should be extracted and modeled as a separate linked individual. 
In general our individuals should be exemplary should model things properly. 
That doesn't mean that we need exhaustively include everything about an individual, but that if we do include it we should model it. Hence I would be just as happy if the text of the purpose were removed completely.
I would also be happy if we recognized that having a sophisticated structure for modeling the goals/purposes of an organization is too onerous and outside the scope of FIBO and eliminated that part of FND.
But if we have examplary information that's in the scope of a FIBO ontology, then I think we're duty bound to use the ontology, not ignore it. If that serves to emphasize how unhelpful that ontology is, then that's a good result.
